content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/product_catalog.rst

```diff
@@ -0,0 +1,107 @@
+===============
+Product catalog
+===============
+
+.. |SO| replace:: :abbr:`SO (sales order)`
+.. |SOs| replace:: :abbr:`SOs (sales orders)`
+.. |BoM| replace:: :abbr:`BoM (bill of materials)`
+.. |RfQ| replace:: :abbr:`RfQ (request for quotation)`
+
+The product *catalog* is a feature integrated with any Odoo app that allows users to add products or
+components to an order. This includes the **Inventory**, **Manufacturing**, **Sales**, **Purchase**,
+and **Repairs** apps, among others.
+
+The product catalog can be accessed from the first tab of a quotation, request for quotation (RfQ),
+order, or bill of materials (BoM) form, and opens in a new page when selected. The catalog displays
```

Comment:
```suggestion
sales order, or bill of materials (BoM) form, and opens in a new page when selected. The catalog displays
```

Reply:
This includes other order types, in addition to sales orders.